pkg/storage/ingest/reader.go

```diff
@@ -881,9 +863,6 @@ func newReaderMetrics(partitionID int32, reg prometheus.Registerer) readerMetric
 			Buckets:                         prometheus.DefBuckets,
 		}),
 		lastConsumedOffset: lastConsumedOffset,
-		kprom: kprom.NewMetrics("cortex_ingest_storage_reader",
-			kprom.Registerer(prometheus.WrapRegistererWith(prometheus.Labels{"partition": strconv.Itoa(int(partitionID))}, reg)),
```

Comment:
Note to reviewers: I removed the "partition" label here. I don't think it was worth it.